Monika After Story/game/script-farewells.rpy

```diff
@@ -1460,27 +1460,110 @@ init 5 python:
     )
 
 label bye_prompt_housework:
-    if mas_isMoniNormal(higher=True):
-        m 1eub "Doing your chores, [player]?"
-        m 1ekc "I would like to help you out, but there's not really much I can do since I'm stuck in here..."
-        m 3eka "Just make sure to come back as soon as you're done, okay?"
-        m 3hub "I'll be waiting here for you~"
-    elif mas_isMoniUpset():
-        m 2esc "Fine."
-        m 2tsc "At least you're doing something responsible."
-        m 2tfc "{cps=*2}...For once.{/cps}{nw}"
+#TODOS: Chance for upset/distressed to not want to go with you, if not I'd ideally like it to default to regular goodbye
+# How to set it to non-generic "we're home" greeting?
+    if persistent.mas_carryme_choice = mas.dockstat.CM_TAKE_MONI
+        if mas_isMoniNormal(higher=True):
+            m "You've got some things to get done around the house?"
+            m "I'm glad you're taking good care of your space, [player]."
+            m "Hang on, I'll come with you!"
+            jump mas_dockstat_iostart
+        elif mas_isMoniUpset():
+            m "Oh, you're doing some work?"
+            m "...Well, I'm glad you're being productive, at least."
+            m "I'm going to make sure you're not slacking off, though."
+            jump mas_dockstat_iostart
+        elif mas_isMoniDis():
```

Comment:
I see in your TODOs you've marked it as a chance for her to offer to come with. I'd say for this we may as well actually leave her coming with you as a normal+ (or maybe upset+ (chance) but nothing below) affection thing. It's easier pathing and imo it makes more sense since she trusts you more.

Reply:
That sounds good to me, and probably makes adding future goodbyes a lot more streamlined.